platform/files.go

```diff
@@ -141,14 +141,14 @@ func (p BuildPlan) Find(bpID string) buildpack.Plan {
 func (p BuildPlan) Filter(metRequires []string) BuildPlan {
 	var out []BuildPlanEntry
 	for _, planEntry := range p.Entries {
-		if !containsEntry(metRequires, planEntry) {
+		if !ContainsEntry(metRequires, planEntry) {
 			out = append(out, planEntry)
 		}
 	}
 	return BuildPlan{Entries: out}
 }
 
-func containsEntry(metRequires []string, entry BuildPlanEntry) bool {
+func ContainsEntry(metRequires []string, entry BuildPlanEntry) bool {
```

Comment:
Why does it need to be public?

Reply:
It doesn't... this is a holdover from an intermediate refactor. I'll change it back.